main.c

```diff
@@ -28,6 +28,8 @@
 #include <llvm-c/Core.h>
 
 #include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
```

Comment:
Looking at https://github.com/sheredom/llvm_bc_parsing_example/pull/1/files#diff-2045016cb90d1e65d71c2407a2570927R39, is `unistd.h` unavailable on Windows?